client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -62,16 +62,26 @@ const MetadataEditor = createClass({
 	},
 
 	handleFieldChange : function(name, e){
+		
+		const trimmedValue = (()=> {
+			if(e.target.value.length === 0) {
+				return e.target.value;
+			}
+			return [...e.target.value.slice(0, -1),e.target.value[e.target.value.length - 1].trim()]; // remove trailing whitespace from last item in list
+		})();
```

Comment:
I think this can be simplified to this:

```
if(e.target.value.length >  0) {
  e.target.value[e.target.value.length - 1] = e.target.value[e.target.value.length - 1].trim();
}
```

Alternatively, we may just want to trim all the values anyway, because you can have the same uniqueness problem if you edit a tag earlier in the list by adding spaces:

```
const trimmedValues = e.target.values.map((str) => str.trim());
```

See I can add a space to the first tag and the value will not be trimmed.
![image](https://github.com/naturalcrit/homebrewery/assets/5878534/2b874b1d-9d56-4925-8ea2-934f5b0f2ad3)